test/tep_app_test.erl

```diff
@@ -0,0 +1,10 @@
+-module(tep_app_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+get_config_argument_valid_test() ->
+    Valid = {ok, [["test.ini", "config.ini"], ["tep.ini"]]},
```

Comment:
I don't understand these `Args`. Is that just a quirk of the `application` behaviour's `init:get_argument` function? In either case, do we document that as a comment or something?

Reply:
Hm ok I just looked up the behaviour of `init:get_argument`. I would have expected tep.ini to be the argument that gets passed to the actual application, because it's the last and "overrides" the others. And also because it doesn't really make sense to pass multiple arguments to the "config file" flag. In fact I'd advocate for us to refuse startup when given multiple config files, but that's pedantic and I'm prepared to drop it. 
Back to the test: The way it's written now is a little obtuse, but I don't know how to well fix it. Could link to docs, but that can sometimes go out of date much faster than anticipated...

Reply:
Since merging of a set of configuration files is a path to hell, I would recommend, not to try this. I think its equal to take the first or the last value. We might discuss whether we stop the startup completely, if multiple `config_file` arguments are found. I'm indifferent about this.